### ISSUE TITLE
Question: Why are both File and FileSet objects being returned in search results

### ISSUE THREAD

**Issue by afred** [WGBH/hydradam2#47]
Modification in #46 allow FileSet objects to be returned in search results, by adding the class name to a clause for the `has_model_ssim` field. However, it's also pulling back `File` objects.

This is not what we want. We want one search result per `FileSet` object. Each result can then display components of the `FileSet` object, like the name of the `original_file`, the `thumbnail`, and technical metadata that was populated via properties extracted from the `technical_metadata` file.

**Comment by afred** [comment]
Example of what's happening:
![image](https://cloud.githubusercontent.com/assets/1507490/12276189/737bc1f8-b942-11e5-9bdf-c421080b7d75.png)

**Comment by afred** [comment]
This may just be a display issue. I grabbed the first result directly out of Fedora, and the returned object was a `FileSet` object, which is what we want.

First verify that all of these are actually FileSet objects like we want.

Then take a closer look into why it is displaying "File" instead of "FileSet" for **ResourceType**.

Worth noting that we actually want to display the word "File", which is more meaningful than "File Set".

**Comment by afred** [comment]
Hmm... i started over from a clean jetty and haven't been able to reproduce. I tried:
  * creating a `Asset => FileSet => File` through the the web UI.
  * creating a `Asset => FileSet => File` through the console.
  * creating just a `FileSet` object through the console.

After all this, doing a blank search returns all results, with either "Asset" or "File" for the Resource Type, which is what we want.

Note that if you click through to the Asset that was created via console, it reports that it contains no files, but that's a different issue I think.

Closing this, and may re-open if the bug resurfaces.
